src/NearOrg/DataAvailabilityPage.jsx

```diff
@@ -550,8 +563,14 @@ return (
               Get started with NEAR's data availability layer
             </Text>
 
-            <Text size="text-l" mobileSize="text-l" style={{ maxWidth: "600px" }}>
-              Efficiently store state data and commitments on a NEAR contract for your L2 rollup.
+            <Text size="text-m" mobileSize="text-m" style={{ maxWidth: "600px" }}>
+              Efficiently store state data and commitments on a NEAR contract
+              for your L2 rollup. Whether you're processing transactions or
+              managing data on the blockchain layer, NEAR's Ethereum-compatible
+              infrastructure offers unparalleled efficiency ensuring seamless
+              interaction with nodes across the network. Explore the power of
+              rollups and experience the future of blockchain technology with
+              NEAR.
```

Comment:
Efficiently store state data and commitments on a NEAR contract for your L2 rollup. Whether you're processing transactions or managing data on the blockchain layer, NEAR's Ethereum-compatible infrastructure offers unparalleled efficiency through our robust network of nodes. Enhance the functionality of your rollup and experience the future of modular blockchain technology with NEAR.

Reply:
resolved by: https://github.com/near/near-discovery-components/pull/744